plugins/woocommerce-admin/client/two-column-tasks/sectioned-task-list.tsx

```diff
@@ -127,13 +131,18 @@ export const SectionedTaskList: React.FC< TaskListProps > = ( {
 		} );
 	};
 
-	const goToTask = ( task: TaskType ) => {
+	const onTaskSelected = ( task: TaskType ) => {
 		trackClick( task );
-		updateQueryString( { task: task.id } );
-	};
+		if ( task.actionUrl ) {
```

Comment:
This was the missing logic, just mimicked what was done in the two column task list.